File: micronaut-hello-world/src/main/java/hello/world/Application.java
package hello.world;

import io.micronaut.runtime.Micronaut;
import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.info.Info;
import io.swagger.v3.oas.annotations.info.License;

@OpenAPIDefinition(info = @Info(
        title = "Hello World",
        version = "0.1",
        description = "HI, MOM!",
        license = @License(
                name = "Public Domain",
                url = "http://unlicense.org")))
public class Application {
    public static void main(final String... args) {
        Micronaut.run(Application.class);
    }
}

File: micronaut-kitchen-sink/src/main/kotlin/hm/binkley/scratch/Application.kt
package hm.binkley.scratch

import io.micronaut.runtime.Micronaut
import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Contact
import io.swagger.v3.oas.annotations.info.Info
import io.swagger.v3.oas.annotations.info.License
import lombok.Generated
import java.time.ZoneOffset.UTC
import java.util.TimeZone

@Generated
@OpenAPIDefinition(
    // TODO: Annotate package, not class
    info = Info(
        contact = Contact(
            name = "B. K. Oxley (binkley)",
            email = "binkley@alumni.rice.edu",
            url = "https://github.com/binkley"
        ),
        description = "Experiments in Micronaut features",
        license = License(
            name = "Public Domain",
            url = "https://unlicense.org/"
        ),
        title = "Micronaut Kitchen Sink",
        version = "0.1"
    )
)
object Application {
    @JvmStatic
    fun main(args: Array<String>) { // TODO: top-level function
        System.setProperty("user.timezone", "UTC")
        TimeZone.setDefault(TimeZone.getTimeZone(UTC))

        Micronaut.build() // TODO: Simplify
            .packages("hm.binkley.scratch")
            .mainClass(Application.javaClass)
            .start()
    }
}
